vignettes/tweak-page-hooks.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: "Custom pkgdown tweak page hooks"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Custom pkgdown tweak page hooks}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+This article will go over the following:
+
+* The process for using custom hooks in R packages
```

Comment:
Is there any documentation around hooks - I have never used them before but now curious.